cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,35 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect -f '{{index .Options "com.docker.network.driver.mtu"}}' bridge
+}
+
+function _patch_calico_manifest() {
+  local -r calico_manifest="$1"
+  local -r calico_diff="$2"
+  local -r destination_dir="$3"
+
+  local -r cri_mtu=$(_get_cri_bridge_mtu)
+  local -r ipip_mode=$(cat "$calico_manifest" | grep CALICO_IPV4POOL_IPIP -A 1 | grep -Po "(Always|Never)")
+  if [ $ipip_mode == "Always" ]; then
+    overhead=$((20))
+    calico_mtu=$((cri_mtu - overhead))
+  else
+    calico_mtu=$(cat "$calico_manifest" | grep -Po "veth_mtu: \"\K\d*")
+  fi
+
+  # Substitute MTU placeholder with the calculated MTU
+  patched_diff=$(mktemp "$destination_dir/calico.XXXX.diff")
+  CNI_MTU=$calico_mtu envsubst < "$calico_diff" > "$patched_diff"
+
+  # Patch the manifest with the patched diff file
+  patched_manifest=$(mktemp "$destination_dir/calico.XXXX.yaml")
+  patch "$calico_manifest" "$patched_diff" -o "$patched_manifest"
+
+  echo "$patched_manifest"
```

Comment:
To dump directly to stdout from `patch` command and patching using stdin
```suggestion
  CNI_MTU=$calico_mtu envsubst < $calico_diff | patch $calico_manifest $patched_diff -o -
```

Reply:
Done